### ISSUE TITLE
[BUG] [Formatter] Docs website is out of date referencing version v1.19.19

### ISSUE THREAD

**Issue by leetrout** [Riverside-Healthcare/djLint#607]
- [ ] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## Issue
```
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/git', 'checkout', 'v1.19.19')
return code: 1
stdout: (none)
stderr:
    error: pathspec 'v1.19.19' did not match any file(s) known to git
Check the log at /home/user/.cache/pre-commit/pre-commit.log
```

## How To Reproduce
1. Open https://www.djlint.com/docs/integrations/
2. Copy the precommit example stanza which currently references version `v1.19.19`

![image](https://user-images.githubusercontent.com/524648/233805518-8bb9d0ff-660e-4b1e-a3cd-d83427f5a832.png)

**Comment by christopherpickering** [comment]
Thanks! Looks like netlify builds were failing... not sure why the latest PR's all "passed" the tests there 😀 

I'll get it updated shortly.
